### ISSUE TITLE
Allow user interaction

### ISSUE THREAD

**Issue by paroxp** [Rory80Hz/rubbernecker#5]
At some point, I'd like to raise a PR, which would allow users to update the stories:

  - Change status/Move card
  - Assign people or themselves

Currently, it's not a great idea, as it would essentially allow everyone to manage our board online. A authentication system would be required...

Happy to discuss further.

**Comment by Rory80Hz** [comment]
Yeah, I did think about this, everyone would have to login, which is doable, but I think we end up over complicating things a bit.

For now, I think the best thing for us to try is to have somebody 'Drive' Pivotal when we are discussing stuff during standup. At the moment, if you need to get to the story in Pivotal it is relatively easy to do so from the links on each one.
